### DISCUSSION TITLE
Is there a schedule or calendar of jams?

### DISCUSSION THREAD

**Discussion by kjcole** [jamulussoftware/jamulus#3034] [General]
I'm very new to Jamulus. In the last few days, I've gotten on twice, just to verify I've got it set up right. (Works okay with my Logi Zone 750 but not my Focusrite Scarlett 2i2. See recently started discussion.)

When I get on, however, the only other people I see are on servers with 80+ ms delay. I haven't figured out when "locals" with a lower latency meet. Is there a good place to look for such info? (I'd be looking for folk and rock, relatively acoustic, as I play flute.)

**Comment by ann0see** [comment]
Hi, Thanks for asking.

I don't think there's a central place - but there is a Facebook group (see the link at the bottom of jamulus.io). Otherwise, there might be some other websites related to the social part - especially WorldJam (which I think is the most organized official Jamulus musical community).
You can always just join a server and ask in the chat.
By your profile, I see you're from the US. I believe there is a community in the US - maybe search in different Genres?

**Reply by kjcole** [reply]
> Hi, Thanks for asking.
> 
> I don't think there's a central place - but there is a Facebook group (see the link at the bottom of jamulus.io). Otherwise, there might be some other websites related to the social part - especially WorldJam (which I think is the most organized official Jamulus musical community). You can always just join a server and ask in the chat. By your profile, I see you're from the US. I believe there is a community in the US - maybe search in different Genres?

Thanks. I just joined the Facebook group. (Skimming, it appears most posts are out of Europe, but I'll keep digging. And keep trying to get my musical friends to try installing Jamulus.)

**Reply by chrisrimple** [reply]
On Facebook, search for the "Jamulus WorldJam" groups.  There are 3 in the USA: PNW, Midwest, and East Coast.  The PNW and Midwest groups are hosting weekly jams (in addition to participating in the WorldJam performances), not sure about East Coast.

**Comment by mcfnord** [comment]
Have you tried running your own server? You might already have the Jamulus server program installed. You could even leave it running for a few days. Maybe someone will wander your way. Also, https://jamulus.live shows local activity first in the list. Whereabouts do you live?
